experimental/torchhub/tests/test_resnet.py

```diff
@@ -13,18 +13,43 @@
 #  limitations under the License.
 
 from pathlib import Path
+from rikai.types.vision import Image
 
 import torchvision
+import pandas as pd
 from pyspark.sql import SparkSession
 
 from rikai.spark.functions import to_image
+from rikai.experimental.torchhub.torchhub_registry import (
+    TorchHubRegistry,
+)
+from rikai.spark.sql.codegen.base import func_from_spec
+
+
+version = f"v{torchvision.__version__.split('+', maxsplit=1)[0]}"
+work_dir = Path().absolute().parent.parent
+image_path = f"{work_dir}/python/tests/assets/test_image.jpg"
+
+
+def test_resnet_model_type():
```

Comment:
This is an attempt to provide mode type utilities. It aims to provide a pyspark-free way to test the model type.